src/components/shared/page.tsx

```diff
@@ -22,15 +22,28 @@ import { insertAdPlaceholders } from 'ads';
 // ----- Components ----- //
 
 const PageStyles = css`
-    ${fontFace("Guardian Egyptian Web", new None, "/public/fonts/egyptiantext-regular.otf")}
-    ${fontFace("Guardian Egyptian Web", new Some(500), "/public/fonts/egyptiantext-medium.otf")}
-    ${fontFace("Guardian Egyptian Web", new Some("bold"), "/public/fonts/egyptiantext-medium.otf")}
-    ${fontFace("Guardian Text Sans Web", new None, "/public/fonts/GuardianSansWeb-Regular.ttf")}
-    ${fontFace("Guardian Headline", new Some(200), "/public/fonts/GHGuardianHeadline-Light.ttf")}
-    ${fontFace("Guardian Headline", new Some(400), "/public/fonts/GHGuardianHeadline-Regular.ttf")}
-    ${fontFace("Guardian Headline", new Some(500), "/public/fonts/GHGuardianHeadline-Medium.ttf")}
-    ${fontFace("Guardian Headline", new Some(700), "/public/fonts/GHGuardianHeadline-Bold.ttf")}
-    ${fontFace("Guardian Icons", new None, "/public/fonts/icons.otf")}
+    ${fontFace("Guardian Egyptian Web", new None, new None, "/public/fonts/GuardianTextEgyptian-Reg.ttf")}
+    ${fontFace("Guardian Egyptian Web", new None, new Some("italic"), "/public/fonts/GuardianTextEgyptian-RegItalic.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some(500), new None, "/public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some("bold"), new None, "/public/fonts/GuardianTextEgyptian-Bold.ttf")}
+    ${fontFace("Guardian Egyptian Web", new Some("bold"), new Some("italic"), "/public/fonts/GuardianTextEgyptian-BoldItalic.ttf")}
+
+    ${fontFace("Guardian Text Sans Web", new Some(200), new None, "/public/fonts/GuardianSansWeb-Light.ttf")}
+    ${fontFace("Guardian Text Sans Web", new Some(300), new None, "/public/fonts/GuardianSansWeb-Regular.ttf")}
+    ${fontFace("Guardian Text Sans Web", new Some(600), new None, "/public/fonts/GuardianSansWeb-Semibold.ttf")}
```

Comment:
Is the `Guardian Sans Web` or `Guardian Text Sans Web`? The filenames suggest it's the former 🤔

Reply:
As Egyptians (serifs) have headline (formerly Egyptian Headline, now Guardian Headline) and text (Egyptian Text) flavours, so do sans serifs:
Guardian Text Sans is text flavour of headline Guardian Sans.

So this mapping isn’t correct: it maps Text to Headline.

Again, grab me if needed.